### ISSUE TITLE
Feature request: Shortcut list

### ISSUE THREAD

**Issue by flofriday** [Moeditor/Moeditor#67]
It would be nice to have a list of all shortcuts (maybe in a new tab at the setting screen) where you can change them, or at least see all shortcuts. 

If you don't like to make a list you can also just write the shortcut to the button like: save (ctr + S)
The save shortcut was not difficult to figure out, but most users won't even know there is a shortcut and I have still now idea how to change the view.

**Comment by tmr232** [comment]
Yeah, an option to view and modify hotkey bindings would be neat.
